au/quantity.hh

```diff
@@ -131,7 +131,7 @@ class Quantity {
     // Construct this Quantity with a value of exactly Zero.
     constexpr Quantity(Zero) : value_{0} {}
 
-    constexpr Quantity() = default;
+    constexpr Quantity() noexcept = default;
```

Comment:
Although it seems like the compiler should have added that specifier, it doesn't seem wrong to be explicit based on this [SO post](https://stackoverflow.com/a/36161478).

Reply:
Thanks for digging!